packages/metrics/modules/create-metric-types/create-metric-types.js

```diff
@@ -18,126 +44,85 @@ const defaultMetricTypesOptions = {
+      buckets: options.buckets || defaultBucketsInSeconds,
     }),
-  };
-
-  /**
-   * NOTE:
-   *    Both metric types should be possible to allow consumers
-   *    to migrate to either or without a metric gap.
-   */
-  if (defaultedOptions.accuracies.includes('ms')) {
-    metrics = {
-      ...metrics,
-      percentilesInMilliseconds:
-        defaultedOptions.metricTypes.includes('summary') &&
-        new Prometheus.Summary({
-          name: defaultedOptions.metricNames.percentilesInMilliseconds,
-          help: 'The HTTP request latencies in milliseconds.',
-          labelNames: defaultRequestLabels
-            .concat(defaultedOptions.labels)
-            .sort(),
-          percentiles: defaultedOptions.percentiles || [
-            0.5,
-            0.9,
-            0.95,
-            0.98,
-            0.99,
-          ],
-        }),
-      bucketsInMilliseconds:
-        defaultedOptions.metricTypes.includes('histogram') &&
-        new Prometheus.Histogram({
-          name: defaultedOptions.metricNames.bucketsInMilliseconds,
-          help: 'The HTTP request latencies in milliseconds.',
-          labelNames: defaultRequestLabels
-            .concat(defaultedOptions.labels)
-            .sort(),
-          buckets: defaultedOptions.buckets || [
-            50,
-            100,
-            300,
-            500,
-            800,
-            1000,
-            1500,
-            2000,
-            3000,
-            5000,
-            10000,
-          ],
-        }),
-    };
-  }
-  if (defaultedOptions.accuracies.includes('s')) {
-    metrics = {
-      ...metrics,
-      percentilesInSeconds:
-        defaultedOptions.metricTypes.includes('summary') &&
-        new Prometheus.Summary({
-          name: defaultedOptions.metricNames.percentilesInSeconds,
-          help: 'The HTTP request latencies in seconds.',
-          labelNames: defaultRequestLabels
-            .concat(defaultedOptions.labels)
-            .sort(),
-          percentiles: defaultedOptions.percentiles || [
-            0.5,
-            0.9,
-            0.95,
-            0.98,
-            0.99,
-          ],
-        }),
-      bucketsInSeconds:
-        defaultedOptions.metricTypes.includes('histogram') &&
-        new Prometheus.Histogram({
-          name: defaultedOptions.metricNames.bucketsInSeconds,
-          help: 'The HTTP request latencies in seconds.',
-          labelNames: defaultRequestLabels
-            .concat(defaultedOptions.labels)
-            .sort(),
-          buckets: defaultedOptions.buckets || [
-            0.05,
-            0.1,
-            0.3,
-            0.5,
-            0.8,
-            1,
-            1.5,
-            2,
-            3,
-            10,
-          ],
-        }),
-    };
-  }
-
-  return metrics;
+});
+
+const createMetricTypes = options => {
+  let defaultedOptions = { ...defaultOptions, ...options };
+
+  var defaultMetrics = getDefaultMetrics(defaultedOptions);
```

Comment:
These could also be consts to communicate intend a bit better, right?

Reply:
I would appriciate that. Either here or I‘ll do it postmerge. However you prefer.